### PR TITLE
Add method to check an integer is a valid HTTP Status Code

### PR DESCRIPTION
**Pull request by wilsonge**
Allows checking you have a valid HTTP Status code. Can be useful when doing things like checking the code associated with exceptions in the App

### CODE REVIEW

File: src/AbstractWebApplication.php [hunk 1]
```diff
@@ -634,6 +634,25 @@ protected function getHttpStatusValue($value)
...
+	 *
+	 * @param   int  $code  The given status as int or string
+	 *
+	 * @return  bool
+	 *
+	 * @since  __DEPLOY_VERSION__
+	 */
+	protected function isValidHttpStatus($code)
+	{
+		if (array_key_exists($code, $this->responseMap))
```
Comment (mbabker) [inline src/AbstractWebApplication.php]: `return array_key_exists($code, $this->responseMap);`

No need to manually specify true/false returns.

File: src/AbstractWebApplication.php [hunk 2]
```diff
@@ -634,6 +634,25 @@ protected function getHttpStatusValue($value)
 	}
 
 	/**
+	 * Check if the value is a valid HTTP 1.1 status code
+	 *
+	 * @param   int  $code  The given status as int or string
```
Comment (zero-24) [inline src/AbstractWebApplication.php]: > The given status as int or string

so we need `string|int` here ;)

File: src/AbstractWebApplication.php (hunk 1 above)
Comment (zero-24) [inline src/AbstractWebApplication.php]: and here a cast to int if it is a string?

File: src/AbstractWebApplication.php (hunk 2 above)
Comment (wilsonge) [inline src/AbstractWebApplication.php]: Or I need to not copy paste the doc block and force an integer :D

Comment (zero-24) [inline src/AbstractWebApplication.php]: hehe :smile:

**Pull request merged**